releasenotes/notes/1.0/estimatorv2-9b09b66ecc12af1b.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features_primitives:
+  - |
+    Added a new :class:`~.BaseEstimatorV2` primitive base class based on
+    `the RFC <https://github.com/Qiskit/RFCs/blob/master/0015-estimator-interface.md>`_.
+    This base class establishes the new ``EstimatorV2`` interface, which adds support for
+    performing vectorized calls so that sweeps over parameter value sets and observables
+    can be efficiently specified.
```

Comment:
We should delete this release note as it's been covered by @ihincks's omnibus primitives v2 release note file.